### PR TITLE
Address feedback

### PR DESCRIPTION
**Pull request by dominicbetts**
Address feedback from @pierreca in pr #9

### CODE REVIEW

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-1.js [hunk 1]
```diff
@@ -18,16 +18,14 @@ console.log('IoT Hub troubleshooting tutorial\nSimulated device #1\n')
...
+// Connect to the IoT hub.
+client.open(function (err) {
   if (err) {
     console.log('Could not connect: ' + err);
   } else {
     console.log('Client connected');
   }
-  client.close();
+  client.close(function() {
   process.exit(0);
```
Comment (pierreca) [inline iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-1.js]: nit: indentation

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js [hunk 2]
```diff
@@ -14,28 +14,42 @@ var Message = require('azure-iot-device').Message;
...
+//
+//   var token = SharedAccessSignature.create(resourceUri, policyName, signingKey, expires);
+//
+//   return token;
+// };
+//
+// // There are two ways to create a SAS token for a device to connect:
 //
 // var token = generateSasToken(
 //  'hubname.azure-devices.net/devices/devicename',
```
Comment (pierreca) [inline iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js]: are both of these meant to be commented out? what's the choice or action that a beginner user should take?

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js [hunk 3]
```diff
@@ -52,21 +66,18 @@ var generateSasToken = function(resourceUri, signingKey, policyName, expiresInMi
...
+// Connect to the IoT hub.
+client.open(function (err) {
   if (err) {
     console.log('Could not connect: ' + err);
   } else {
     console.log('Client connected');
   }
-  client.close();
+  client.close(function() {
   process.exit(0);
```
Comment (pierreca) [inline iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js]: nit: indentation

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js [hunk 4]
```diff
@@ -25,8 +25,15 @@ function printResultFor(op) {
...
+client.open(function (err) {
+  if (err) {
+    console.log(chalk.red('Could not connect: ' + err));
+  } else {
+    console.log('Client connected');
+
+    // Listen for TestMethod being called from the hub
+    client.onDeviceMethod('TestMethod', function(request, response) {
   // Function to send a direct method reponse to your IoT hub.
   function directMethodResponse(err) {
```
Comment (pierreca) [inline iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js]: nit: indentation

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js [hunk 5]
```diff
@@ -41,17 +48,7 @@ function onTestMethod(request, response) {
...
-// Callback function to run after connecting to the IoT hub.
-var connectCallback = function (err) {
-  if (err) {
-    console.log(chalk.red('Could not connect: ' + err));
-  } else {
-    console.log('Client connected');
-
-    // Listen for TestMethod being called from the hub
-    client.onDeviceMethod('TestMethod', onTestMethod);
+    });
```
Comment (pierreca) [inline iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js]: I think this should be:
```js
client.onDeviceMethod('TestMethod', function(request, response) {
  // request contains the payload of the method call
  // response is the object used to send the result
  response.send(200, 'TestMethod received: ' + JSON.stringify(request.payload), function (err) {
    if (err) {
      console.error('Could not send method response: ' + err.toString());
    } else {
      console.log('Method response sent');
    }
  });
});
```

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-1.js (hunk 1 above)
Comment (dominicbetts) [reply iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-1.js]: Fixed

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js (hunk 2 above)
Comment (dominicbetts) [reply iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js]: This commented code is referenced from the tutorial article.

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js (hunk 3 above)
Comment (dominicbetts) [reply iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-2.js]: Fixed

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js (hunk 4 above)
Comment (dominicbetts) [reply iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js]: Fixed

File: iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js (hunk 5 above)
Comment (dominicbetts) [reply iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js]: Fixed

**Pull request merged**